src/mon/MonCommands.h

```diff
@@ -1161,7 +1161,7 @@ COMMAND("mgr dump "
 	"name=epoch,type=CephInt,range=0,req=false",
 	"dump the latest MgrMap",
 	"mgr", "r")
-COMMAND("mgr fail name=who,type=CephString",
+COMMAND("mgr fail name=who,type=CephString,req=false",
```

Comment:
Did this sneak in accidentally?  This doesn't look like it has anything to do with ptrace